src/utils/logger.ts

```diff
@@ -0,0 +1,21 @@
+import winston from 'winston';
+
+const logger = winston.createLogger({
+  level: 'info',
+  transports: [
+    process.env.NODE_ENV !== 'production'
```

Comment:
Could we use feature flag instead which is not set in production? benefit is also that if you locally want to disable logging you can do it. I have felt that it's harder now to debug api-calls because logging creates so much noise :)